packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -912,16 +912,13 @@ export class DeltaManager
             this.timeTillThrottling = timeNow + delaySeconds;
 
             // Add 'throttling' properties to an error with safely extracted properties:
-            const throttlingWarning: IThrottlingWarning = {
+            const reconfiguredError = CreateContainerError(error);
+            reconfiguredError.addTelemetryProperties({
                 errorType: ContainerErrorType.throttlingError,
```

Comment:
does this always work correctly? if reconfiguredError already has error type, will these properties overwrite it? or will the original property win.